dask/dataframe/io/tests/test_json.py

```diff
@@ -13,6 +13,74 @@
 ddf = dd.from_pandas(df, npartitions=2)
 
 
+@pytest.mark.parametrize("orient", ["split", "records", "index", "columns", "values"])
+def test_read_json_with_path_column(orient):
+    with tmpfile("json") as f:
+        df.to_json(f, orient=orient, lines=False)
+        actual = dd.read_json(f, orient=orient, lines=False, include_path_column=True)
+        actual_pd = pd.read_json(f, orient=orient, lines=False)
+        actual_pd["path"] = pd.Series((str(f),) * len(actual_pd), dtype="category")
+        out = actual.compute()
+        assert_eq(out, actual_pd)
```

Comment:
We tend to not call `compute()` on Dask collections in favor of passing them directly to `assert_eq` since `assert_eq` will both call `compute()` internally as well as perform a few other sanity checks on Dask objects

```suggestion
        assert_eq(actual, actual_pd)
```

Reply:
Other tests in this file use `compute()`. Should I change those as well, or just leave it?

Reply:
We should change those as well